scripts/utils.js

```diff
@@ -0,0 +1,35 @@
+require("dotenv").config()
+
+module.exports = {
+  getRealmNetworkFromArgs,
+  getRealmNetworkFromNetwork,
+}
+
+function getRealmNetworkFromArgs () {
+  let realm = process.env.WITNET_EVM_REALM
+    ? process.env.WITNET_EVM_REALM.toLowerCase()
+    : "default"
+  let network = "development"
+  const args = process.argv.join("=").split("=")
+  const networkIndex = args.indexOf("--network")
+  if (networkIndex >= 0) {
+    network = args[networkIndex + 1]
+    realm = (network || "default").split(".")[0]
+  }
+  if (realm === "ethereum") realm = "default"
+  return [realm, network]
+}
```

Comment:
For more consistency, unified handling of defaults and reduced code repetition:

```suggestion
function getRealmNetworkFromArgs () {
  // Try to extract realm/network info from environment
  let network = process.env.WITNET_EVM_REALM
    ? process.env.WITNET_EVM_REALM.toLowerCase()
    : "default"
  // If a `--network` argument is provided, use that instead
  const args = process.argv.join("=").split("=")
  const networkIndex = args.indexOf("--network")
  if (networkIndex >= 0) {
    network = args[networkIndex + 1]
  }

  // Tell realm and network from one another
  return getRealmNetworkFromNetwork(network)
}
```

This is a common pattern that I really love: allowing the impure function (`getRealmNetworkFromArgs`) to only handle the side-effects and then delegate the functional business logic to a pure function (`getRealmNetworkFromNetwork`).

Reply:
Nice. Moreover, there was still some logic missing: 
- What if WITNET_EVM_REALM is set and no prefix in network string? Add WITNET_EVM_REALM value as prefix. 
- What if WITNET_EVM_REALM is set, but network string refers to a different realm? Catch and throw.